### PR TITLE
style(frontend): fix social icons rendering

### PR DESCRIPTION
**Pull request by BonomoAlessandro**
# Motivation

The so social icons on the dappmodal do not have the same size which looks bad. It is uncommon that github is displayed before X, so it needs to be swapped.

# Changes

- fixe size of social icons on dappmodal

# Tests

before:

dAppModal:
image

after:

dAppModal:
image

### CODE REVIEW

File: src/frontend/src/lib/components/icons/IconGitHub.svelte [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
+			fill="currentColor"
+		/>
+	</g>
+	<defs>
+		<clipPath id="clip0_2345_9164">
+			<rect width="24" height="24" fill="white" />
+		</clipPath>
+	</defs>
```
Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/lib/components/icons/IconGitHub.svelte:18]: what is this additional `defs` part?

File: src/frontend/src/lib/components/icons/IconTwitter.svelte [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
+			clip-rule="evenodd"
+			d="M15.9455 23L10.396 15.0901L3.44886 23H0.509766L9.09209 13.2311L0.509766 1H8.05571L13.286 8.45502L19.8393 1H22.7784L14.5943 10.3165L23.4914 23H15.9455ZM19.2185 20.77H17.2398L4.71811 3.23H6.6971L11.7121 10.2532L12.5793 11.4719L19.2185 20.77Z"
+			fill="currentColor"
+		/>
+	</g>
+	<defs>
+		<clipPath id="clip0_2345_9171">
+			<rect width="24" height="24" fill="white" />
+		</clipPath>
+	</defs>
```
Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/lib/components/icons/IconTwitter.svelte:18]: ditto

**Comment by peterpeterparker** [comment]
For or future contributions, it would have been nice to split this into two PRs. One topic was the size of the dApp explorer icons, and the other was the order of OISY's social icons—two distinct concerns. Just thinking at hopefuly someday we got more E2E tests and screenshots running.

File: src/frontend/src/lib/components/icons/IconGitHub.svelte (hunk 1 above)
Comment (BonomoAlessandro) [reply src/frontend/src/lib/components/icons/IconGitHub.svelte:18]: You are right, the `defs` part is not used

**Comment by BonomoAlessandro** [comment]
@AntonioVentilii-DFINITY pls review

**Comment by AntonioVentilii** [comment]
@BonomoAlessandro before review, may you please follow @peterpeterparker suggestion and separate the order of social icons in another PR?

**Comment by BonomoAlessandro** [comment]
@AntonioVentilii done. The order of the social icons is now separated into antother PR.

**Review by AntonioVentilii — APPROVED** [review]
LGTM tks!

**Pull request merged**